src/main/java/frc/robot/vision/rawdata/RawAprilTagVisionData.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.vision.rawdata;
+
+import edu.wpi.first.apriltag.AprilTag;
+import edu.wpi.first.math.geometry.Pose3d;
+import frc.robot.poseestimator.observations.IRobotPoseVisionObservation;
+
+public class RawAprilTagVisionData extends RawVisionData implements IRobotPoseVisionObservation {
```

Comment:
The relationship between the classes and the interface here is not that clear, we have an interface, but it is separated between 2 classes. How will you use each one?

Reply:
Can I resolve it since `IRobotPoseVisionObservation` [got deleted](https://github.com/GreenBlitz/GB-Robot-Template/pull/498/commits/b1e19bbcd96986dc62e2af3bffc505137a23d007)?